testing/trino-product-tests/pom.xml

```diff
@@ -171,6 +186,12 @@
             <artifactId>testng</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
I would be strongly in favor of putting the test resources in a separate project and not in `trino-testing` . `trino-product-test` has no need currently other than the test resources for `trino-testing`.

@findepi

Reply:
I am fine with separate project for delta test resources. I think this is how we had this internally btw.